setup.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
 from pydicom import __version__
-
+from glob import glob
+import os
```

Comment:
Change the order

```python
import os
from glob import glob
 from setuptools import setup, find_packages

 from pydicom import __version__
```

refer to PEP8:

mports should be grouped in the following order:

    standard library imports
    related third party imports
    local application/library specific imports

You should put a blank line between each group of imports.